picard/const/__init__.py

```diff
@@ -35,6 +35,7 @@
 )
 
 USER_PLUGIN_DIR = os.path.join(USER_DIR, "plugins")
+USER_DOWNLOADS_DIR = os.path.join(USER_DIR, "downloads")
```

Comment:
Doesn't this mean that we store each file twice on disk - once in the network cache (where it will eventually be cleaned up) and once in USER_DOWNLOADS_DIR(where no cleanup happens)? If we store the zip files permanently, why don't we import the plugins directly from them instead of extracting the zip file, creating a third copy of the contents?